hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -353,14 +353,20 @@ private static int deduplicatePartitionPath(JavaSparkContext jsc, String duplica
 
   private static int doBootstrap(JavaSparkContext jsc, String tableName, String tableType, String basePath,
       String sourcePath, String recordKeyCols, String partitionFields, String parallelism, String schemaProviderClass,
-      String bootstrapIndexClass, String selectorClass, String keyGeneratorClass, String fullBootstrapInputProvider,
+      String bootstrapIndexClass, String selectorClass, String keyGenerator, String fullBootstrapInputProvider,
       String payloadClassName, String enableHiveSync, String propsFilePath, List<String> configs) throws IOException {
 
     TypedProperties properties = propsFilePath == null ? UtilHelpers.buildProperties(configs)
         : UtilHelpers.readConfig(FSUtils.getFs(propsFilePath, jsc.hadoopConfiguration()), new Path(propsFilePath), configs).getConfig();
 
     properties.setProperty(HoodieBootstrapConfig.BOOTSTRAP_BASE_PATH_PROP, sourcePath);
-    properties.setProperty(HoodieBootstrapConfig.BOOTSTRAP_KEYGEN_CLASS, keyGeneratorClass);
+    if (!StringUtils.isNullOrEmpty(keyGenerator)) {
+      if (keyGenerator.startsWith("org.apache.hudi")) {
```

Comment:
Can we make sure, whether all the customized key generators start with `org.apache.hudi`?

Reply:
> Can we make sure, whether all the customized key generators start with `org.apache.hudi`?

we can

Reply:
> Can we make sure, whether all the customized key generators start with `org.apache.hudi`?

@yanghua  Fixed, PTAL